src/frontend/Sider/Personoversikt/OpprettNyBehandlingModal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+
+import { styled } from 'styled-components';
+
+import { Button, Select, VStack } from '@navikt/ds-react';
+
+import KlageGjelderTilbakekreving from './KlageGjelderTilbakekreving';
+import { useApp } from '../../context/AppContext';
+import { Feilmelding } from '../../komponenter/Feil/Feilmelding';
+import { ModalWrapper } from '../../komponenter/Modal/ModalWrapper';
+import DateInput from '../../komponenter/Skjema/DateInput';
+import { RessursStatus } from '../../typer/ressurs';
+
+export const DatoWrapper = styled.div`
+    margin-top: 1rem;
+`;
+
+interface OpprettKlageRequest {
+    mottattDato: string;
+    klageGjelderTilbakekreving: boolean;
+}
+
+const OpprettNyBehandlingModal = () => {
+    const { request } = useApp();
+
+    const [visModal, settVisModal] = useState(false);
+    const [nyBehandlingValg, settNyBehandlingValg] = useState('');
```

Comment:
Kanskje kan kalle for `Behandlingstype` og at det er en enum?

Reply:
Her fant jeg en allerede eksisterende enum (BehandlingType), så jeg brukte denne isteden. Se gjerne på TODO som jeg har lagt inn: Er det nødvendig med: `event.persist();` ?

commit: 055e7a9f18263e7e4de510e9655b156ad7cf7a11

Reply:
Persist kan fjernes 
> As of v17, e.persist() doesn’t do anything because the SyntheticEvent is no longer [pooled](https://legacy.reactjs.org/docs/legacy-event-pooling.html).

Reply:
Fjernet: bc8cabd50149eab1fcea6797657354d363532436